tests/src/Functional/GraphQLFunctionalTestBase.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Drupal\Tests\graphql\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\graphql\Traits\GraphQLFunctionalTestsTrait;
+use Drupal\Tests\graphql\Traits\QueryFileTrait;
+
+/**
+ * The base class for all functional GraphQL tests.
+ */
+abstract class GraphQLFunctionalTestBase extends BrowserTestBase {
```

Comment:
we have a HttpRequestTrait that we use in the kernel tests, can we use that one to test the request/response workflow? Maybe we need to enhance it a bit?

Then we would not need to invent a new function test category here.